src/elementary.hpp

```diff
@@ -11,3 +11,5 @@
 // free functions to produce new identifiers
 #include "elementary/src/absorb.hpp"
 #include "elementary/src/emit.hpp"
+#include "elementary/src/fromEndfReactionNumber.hpp"
+#include "elementary/src/toEndfReactionNumber.hpp"

```

Comment:
I didn't see `toEndfReactionNumber.hpp` in this. Also, the Style Guide prefers `toENDFReactionNumber` instead of `toEndfReactionNumber`.

Reply:
Both files are there. toEndfReactionNumber is already in master.

Reply:
Ah, yes. I was wondering why this would compile when it looked like files may have been missing.